fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Utils.java

```diff
@@ -160,27 +160,28 @@ public static List<ColumnRefOperator> extractColumnRef(ScalarOperator root) {
         }
 
         for (ScalarOperator child : root.getChildren()) {
-            list.addAll(extractColumnRef(child));
+            if (list.isEmpty()) {
+                list = (LinkedList<ColumnRefOperator>) extractColumnRef(child);
```

Comment:
use `List<ColumnRefOperator> list =  new LinkedList<>()`, then remove the if branch